toolbox/linux/why.rst

```diff
@@ -0,0 +1,54 @@
+Why Use Linux?
+==============
+
+Nobody really likes change, and a new operating system means a lot of change.
+Why do we think you should learn it?  Here's a short list:
+
+It's Easier to do Programming
+-----------------------------
+
+Apple makes iOS and OS X development very easy on Macs. Microsoft makes Windows
+development very easy on PCs. However, web development using the most popular
+tools is easiest on Linux, and so is cross platform development. The main reason
+for this is that Linux treats the command line as a "first-class" tool.
+Everything you can do with a GUI [#fn-gui]_ you can also do on the command line,
```

Comment:
Why a footnote here when we can just make the text "GUI" be a link to the Wikipedia page?